src/repology/repology_cli.py

```diff
@@ -43,8 +43,14 @@ def _pkg_str(str_obj):
     raise ArgumentTypeError("Value must be a non-empty string")
 
 
-def getargs():
-    """Parse command line arguments"""
+def getargs(str_args=None):
```

Comment:
I think this would be cleaner if it accepted `args`, which can be a list of strings.

 - There's nothing restricting us to a string in python
 - We currently split that string into an array in L100 anyways.
 - That'd fix various cases where the string splitting currently would be wrong, as it's not escaping-aware.

Reply:
Fixed in https://github.com/tiiuae/sbomnix/commit/ecf9b87d14ceb3b3668dc5b05746fc2b754d8195